native/core/src/executor.rs

```diff
@@ -101,7 +101,6 @@ impl Executor {
             }
             Ok(keys) => {
                 trace!(?event_id, "handling done");
-                info!("******************** executor handled: {:?}", keys.clone());
```

Comment:
why are you removing all of these? They are useful for debugging, which is the reason they are thre? Arguably the "***" can be removed but why take them out entirely?